native-oci-generative-ai/pom.xml

```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.oracle.labs</groupId>
+  <artifactId>generative-ai</artifactId>
+  <version>1.0-SNAPSHOT</version>
+  <name>Oracle Cloud Infrastructure - Generative AI Demo</name>
+  <description>This project contains an example of using Oracle Cloud Infrastructure to generate text.</description>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.compiler.source>21</maven.compiler.source>
+    <maven.compiler.target>21</maven.compiler.target>
```

Comment:
Is there a technical reason to lock the Java version? I think we can just leave it out so it's using the Java version of the `JAVA_HOME` provided:
```suggestion
```